src/main/app/case/case.ts

```diff
@@ -140,6 +140,16 @@ export interface Case {
   childrenFirstNameAfterAdoption?: string;
   childrenLastNameAfterAdoption?: string;
 
+  otherParentAddressKnown?: YesOrNo;
+  otherParentFirstNames?: string;
+  otherParentLastNames?: string;
+  otherParentExists?: YesOrNo;
+  otherParentAddress1?: string;
+  otherParentAddress2?: string;
+  otherParentAddressTown?: string;
+  otherParentAddressCounty?: string;
+  otherParentAddressPostcode?: string;
+
```

Comment:
Other parent can have international address, hence we need more fields for international address
Refer to birth mother address fields